tests/ast/data/df_first.test

```diff
@@ -16,4 +16,8 @@ res2 = res1.first(-5, True)
 
 res3 = res1.first(2, True)
 
-res4 = res1.first(1, True)
+res4 = res1.limit(2, 0)
```

Comment:
This seems wrong. There was no `limit` call in the test case.

Reply:
fixed, sorry. Forgot to use the suppress ast parameter